tsconfig.json

```diff
@@ -12,10 +12,5 @@
         // "noFallthroughCasesInSwitch": true, /* Report errors for fallthrough cases in switch statement. */
         // "noUnusedParameters": true,  /* Report errors on unused parameters. */
     },
-
-    "include": ["src", "test"],
-    "references": [
-        { "path": "./lsp/client" }, 
-        { "path": "./lsp/server" }
-    ]
+    "include": ["src", "test", "test/suite/lsp/server/testFixture/**/*.json"]
```

Comment:
Can we set `rootDir` back to `src` as well? Lost that change in https://github.com/salesforce/salesforcedx-vscode-mobile/pull/155, and it's why we now have an `out/src` folder structure in our build.

Reply:
Yes, it was done intentionally because `test` folder now sits at the same level with `src`. I wanted to have these two folders reflect their same positions under `out` as well so that I can run tests and then to explicitly not include the test folder in vsix package(through .vscodeignore).